test/dsl/dsl_advanced_model_construction.jl

```diff
@@ -212,8 +212,8 @@ end
 
 # Checks that repeated metadata throws errors.
 let 
-    @test_throws LoadError @eval @reaction k, 0 --> X, [md1=1.0, md1=2.0] 
-    @test_throws LoadError @eval @reaction_network begin
```

Comment:
This was never supposed to be a `LoadError` in the first place. There was a variable in the `error` statement that was undeclared, so that yielded an `LoadError`, which was not the true error.